### PR TITLE
[C-2429] Remove hls

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Removes hls now that we exclusively use mp3
- Tested on tracks/albums/offline/collectible-playlists
- Tested on web and mobile

### CODE REVIEW

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-2429-remove-hls

File: packages/common/src/utils/hls.test.js [hunk 1]
```diff
@@ -1,112 +0,0 @@
-import { generateM3U8, generateM3U8Variants } from './hls'
```
Comment (raymondjacobson) [inline packages/common/src/utils/hls.test.js:1]: rip. we should have more tests

File: packages/web/scripts/publishScripts.sh [hunk 2]
```diff
@@ -9,13 +9,11 @@ make -C "./node_modules/exif-parser" build-browser-bundle
 declare -a modules=(
   "./node_modules/jimp/browser/lib/jimp.min.js"
   "./node_modules/exif-parser/dist/exif-parser-0.1.12-min.js"
-  "./node_modules/hls.js/dist/hls.min.js"
```
Comment (raymondjacobson) [inline packages/web/scripts/publishScripts.sh:12]: i guess this was no longer used

**Review by raymondjacobson — APPROVED** [review]

**Pull request merged**